docs/reference/method/MongoDBClient-startSession.txt

```diff
@@ -0,0 +1,80 @@
+===============================
+MongoDB\\Client::startSession()
+===============================
+
+.. default-domain:: mongodb
```

Comment:
Can we add `.. versionadded:: 1.3` above this (separated by a empty line on top and bottom)?